src/header/header.scss

```diff
@@ -190,181 +136,191 @@
-        transition: all $iui-speed-fast ease;
-      }
-      @include themed {
-        opacity: t(iui-opacity-3);
-      }
+    @media (max-width: 768px) {
+      margin-left: 0;
     }
   }
 
   > .iui-button-icon {
     flex-shrink: 0;
+    box-sizing: content-box;
   }
 
   > * + * {
     margin-left: $iui-sm;
   }
 
   .iui-header-button-icon {
+    display: inline-flex;
     padding: $iui-xs;
+    box-sizing: content-box;
     @include iui-header-button-icon($iui-icons-default, $iui-border-radius);
     @include themed {
       background-color: t(iui-color-background-3);
       fill: t(iui-icons-color);
     }
+
+    @media (max-width: 768px) {
+      display: none;
+    }
+  }
+
+  img.iui-header-button-icon {
+    padding: 0;
+    width: auto;
+    height: $iui-icons-default + ($iui-xs * 2);
   }
 
   &.iui-active {
     @include iui-header-button-active;
   }
 
-  &:disabled {
+  &.iui-disabled,
+  &:disabled,
+  &[aria-disabled='true'] {
+    cursor: not-allowed;
+
     .iui-header-button-icon {
       filter: grayscale(1) opacity(0.7);
     }
   }
 
   @at-root {
-    .iui-page-header.iui-slim {
-      > .iui-left {
-        .iui-button {
-          .iui-header-button-icon.iui-header-button-icon {
-            width: $iui-icons-small;
-            height: $iui-icons-small;
-            padding: $iui-xxs;
-          }
+    .iui-slim {
+      .iui-header-breadcrumbs-button {
+        .iui-header-button-icon {
+          width: $iui-icons-small;
+          height: $iui-icons-small;
+          padding: $iui-xxs;
+          margin: 0 $iui-xs;
+        }
+
+        img.iui-header-button-icon {
+          width: $iui-icons-small + ($iui-xxs * 2);
+          height: $iui-icons-small + ($iui-xxs * 2);
+          padding: 0;
         }
 
-        .iui-button-label {
-          .iui-description {
-            @include iui-display-none;
-          }
+        .iui-header-breadcrumbs-button-description {
+          @include iui-display-none;
         }
       }
     }
   }
 }
 
+@mixin iui-header-button {
+  @include iui-button;
+  @include iui-button-borderless;
+  @include iui-header-button-reset;
+  flex-shrink: 0;
+}
+
+@mixin iui-header-button-icon($size: $iui-icons-large, $border-radius: $iui-border-radius) {
+  width: $size;
+  height: $size;
+  border-radius: $border-radius;
+  @media (prefers-reduced-motion: no-preference) {
+    transition: width $iui-speed-fast ease-out, height $iui-speed-fast ease-out;
+  }
+}
+
 @mixin iui-header-split-button {
+  max-width: 25vw;
+  transition: background-color $iui-speed-fast ease-out;
+
   &,
   > * {
     height: 100%;
     display: flex;
     overflow: hidden;
   }
 
-  max-width: 25vw;
-
-  .iui-button {
-    @include iui-header-button;
+  .iui-header-breadcrumbs-button {
     padding: 0 $iui-xs;
   }
 
   > *:first-child {
-    .iui-button {
+    .iui-header-breadcrumbs-button {
       padding-left: $iui-sm;
+
+      &:hover {
+        background-color: transparent;
```

Comment:
Is this needed? cause now hover on projects button and imodels button is different by 0.1 opacity :DD

Reply:
If I remove this line we get a double hover effect on the left part of a split button:
![Screen Recording 2022-02-18 at 4 43 40 PM](https://user-images.githubusercontent.com/849817/154765430-0c6d7914-7448-4775-a59b-15faeb636a4d.gif)

Reply:
But isnt it how it works now??

Reply:
That is not how it is now.  In our current implementation if you hover over the left part of the split button you do NOT get a double hover effect on the left portion.  See this GIF and compare Project Alpha's hover states:
![Screen Recording 2022-03-01 at 4 16 46 PM](https://user-images.githubusercontent.com/849817/156250423-bb9e255d-3901-4538-8c6f-b9947d630750.gif)